portal-api/api-tests/v1/utilisation-report-service/previous-reports.api-test.js

```diff
@@ -7,7 +7,7 @@ const { withClientAuthenticationTests } = require('../../common-tests/client-aut
 const { withRoleAuthorisationTests } = require('../../common-tests/role-authorisation-tests');
 const { PAYMENT_REPORT_OFFICER } = require('../../../src/v1/roles/roles');
 
-const collectionName = 'utilisation-reports';
+const collectionName = 'utilisationReports';
```

Comment:
Could we add a `DB_COLLECTIONS` const to the `portal-api` project to use here instead (and the next file too)?